src/App.tsx

```diff
@@ -1,8 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p>Nothing was pressed yet</p>
-    <p>The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | null,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: null,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleDocumentKeyUp);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleDocumentKeyUp);
+  }
+
+  handleDocumentKeyUp = (event: KeyboardEvent) => {
+    this.setState({
+      pressedKey: event.key,
+    });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        {this.state.pressedKey ? (
+          <p>{`The last pressed key is ${this.state.pressedKey}`}</p>
+        ) : (
+          <p>Nothing was pressed yet</p>
```

Comment:
It's good to add condition when key is space. Becouse nothing heppens when you press it